### PR TITLE
config split USA

### PR DESCRIPTION
**Pull request by frodrigo**
- Florida #1696
- Illinois #1697
- Georgia #1698
- Noth Carolina #1699

### CODE REVIEW

File: osmose_config.py [hunk 1]
```diff
@@ -975,7 +1001,15 @@ class gen(default_country):
 us_ny_region("southern_tier", (1838015,1838025,1837994,1837996,1837991,1838024,1838014,1838017))
 us_ny_region("western_new_york", (68855,36267,78970,1837997,36074,1837989,1838089,78971))
 
-us_state("north-carolina", 224045, "US-NC", proj=3631)
+us_nc_region = gen_country('north-america/us-south/north-carolina', country_base='usa_north_carolina', country_code="US-NC", download_repo=OSMFR, language='en', proj=3631)
+us_nc_region("north_central", (2528685,2528696,2528698,2528700,2528703,2528707,2528713,2528715,2528726,2528730,2528733,2528751,2528697,2528752,2528757))
+us_nc_region("northest", (2528675,1242005,1242002,1242006,1232243,1232267,1242003,2528706,1242000,1236353,2528720,2528728,1242004,1242001,2528734,2528749,2528753))
```
Comment (Famlam) [inline osmose_config.py:1006]: Do you mean northeast?

Comment (frodrigo) [reply osmose_config.py]: Yes. Thank you, fixed.

File: osmose_config.py [hunk 2]
```diff
@@ -936,11 +936,38 @@ class gen(default_country):
 us_state("connecticut", 165794, "US-CT", proj=3507)
 us_state("delaware", 162110, "US-DE", proj=3509)
 us_state("district-of-columbia", 162069, "US-DC", proj=3559)
-us_state("florida", 162050, "US-FL", proj=3513)
-us_state("georgia", 161957, "US-GA", proj=26917)
+
+us_fl_region = gen_country('north-america/south/florida', country_base='usa_florida', country_code='US-FL', download_repo=OSMFR, language='en', proj=3513)
```
Comment (jocelynj) [inline osmose_config.py:940]: To align with geofabrik, it should be us-south, not south.

File: osmose_config.py [hunk 3]
```diff
@@ -936,11 +936,38 @@ class gen(default_country):
...
+us_fl_region("gatorland", (1210739,1210741,1210721,1210723,1210708,1210752))
+us_fl_region("gold_coast", (1210714,1210692,1210746,1210746,1210748))
+us_fl_region("northeast", (1210735,1210713,1210715,1210733,1210719,1210749,1210720,1210689,1210745))
+us_fl_region("northwest", (1210709,1210712,1210697,1210742,1210732,1210699,1210736,1210693,1210685,1210730,1210703,1210717))
+us_fl_region("panhandle", (1210718,1210743,1210737,1210694,1210750,1210687,1210691,1210706,1210716,1210751))
+us_fl_region("southwest", (1210722,1210705,1210727,1210744,1210725))
+us_fl_region("suncoast", (1210740,1210695,1210738,1210700,1210702,1210701,1210690,1210707,1210726,1214525,1210734))
+us_fl_region("treasure_coast", (1210711,1210698,1210747,1210696))
+
+us_ga_region = gen_country('north-america/south/georgia', country_base='usa_georgia', country_code='US-GA', download_repo=OSMFR, language='en', proj=26917)
```
Comment (jocelynj) [inline osmose_config.py:951]: -> us-south

File: osmose_config.py [hunk 4]
```diff
@@ -936,11 +936,38 @@ class gen(default_country):
...
+us_ga_region("southeast", (966037,966038,966043,1020678,1025644,1027983,1028132,1028123,1028129,416911,417391,1059839,417302,417316,1059835,1073783,447655,1073771,1073778,413939,1073769,1028130,1073870,1028120,1073866,1073767,1073784,1073865,1073775,1073862,1027989,1027985,1073868,1073861,1073781,1073926,1073923,1073930,1073922,1073773,420422,416868,1073927,1073770,1074322,1074318,420860,1074321,1074317,1028126,1074320,1073768,1027980))
+us_ga_region("southwest", (1020676,1027990,1027981,1027982,1027982,11739369,1028122,1059842,1028128,1027986,1073774,1027987,1073765,1073782,1073788,1028119,1028125,1073776,1074368,1073766,1073766,1073772,1059838,1073763,1073779,1073925,1074381,1073928,1028133,1073924,1073764,1059840,1073780,1074319,1027984,1073787,1073777,1027979,1074324,1073785))
+
 us_state("hawaii", 166563, "US-HI", proj=2783) # note: projection for hawaii is the one used for center islands, not for the whole
 us_state("idaho", 162116, "US-ID", proj=3741)
-us_state("illinois", 122586, "US-IL", proj=3746)
+
+us_il_region = gen_country('north-america/center/illinois', country_base='usa_illinois', country_code='US-IL', download_repo=OSMFR, language='en', proj=3746)
```
Comment (jocelynj) [inline osmose_config.py:961]: center -> us-midwest

**Comment by jocelynj** [comment]
Thanks, I've added the poly files to http://download.openstreetmap.fr/polygons/north-america/

Extracts will be generated in a few days.

**Comment by jocelynj** [comment]
Extracts are available.

**Pull request merged**